README.md

```diff
@@ -15,12 +15,14 @@ NOTHING here is final.
 > ├── `main.py` &/or `start_bot.py`  
 > ├── `setup.py`  
 > ├── `.gitignore`  
-> ├── `Data/`  
+> ├── `requirements.txt`
+> │── `changelog.txt`  
+> ├── `infrastructure/`  
 > │    ├── `Dockerfile`  
 > │    ├── `docker-compose.yml`  
-> │    ├── `changelog.txt`  
-> │    ├── `requirements.txt`  
+> ├── `data/`  
 > │    └── `historical_data/`  
```

Comment:
Edit: This issue can be solved by placing historical data under `Data/historical_data/{version}`.

This was also unrelated to the PR. My bad!

Reply:
You see the point ^^ each change can have multiple solution and so discussions, we cannot do the refactoring in only one PR.
I'm not comfortable if the `version` path because the schema of historical data may not changed between multiple version.
The best way for me is to not change anything until we have an issue about it.

Reply:
Got some feedback from @enarjord, will move each of these issues into separate threads.